pkg/apis/cassandra.rook.io/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -36,6 +37,7 @@ func (in *Cluster) DeepCopyInto(out *Cluster) {
 }
 
 // DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new Cluster.
+// #nosec G601/
```

Comment:
Instead of adding this to every DeepCopy method, can we add it to the top of the file to apply to the whole file?

Reply:
yes, we can do that.